buddyboss/buddyboss-platform
# PROD-878 Fix #1939: Fix Link preview issue in activity feed

### All Submissions:


### Changes proposed in this Pull Request:

Fixes #1939

### How to test the changes in this Pull Request:

1. Enable link preview from wp-admin > BuddyBoss > Settings > Activity 
2. Insert some text in the global or personal activity feed entry form
3. Add a link to some text and press the enter button to see the link preview
4. video from issue creator - http://somup.com/cYl3lf2HHS

### Proof Screenshots or Video

https://www.loom.com/share/7bf3c38fb5de4d2a850719d6ac9353bd

### Other information:
